core/controllers/story_editor.py

```diff
@@ -138,3 +138,19 @@ def put(self, story_id):
             topic_services.unpublish_story(topic_id, story_id, self.user_id)
 
         self.render_json(self.values)
+
+
+class ValidateExplorationsHandler(base.BaseHandler):
+    """A data handler for validating the explorations in a story."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_edit_story
+    def get(self, _, exp_ids):
+        """Validation issues for explorations in the story."""
+        exp_ids = exp_ids.split(',')
```

Comment:
Probably better to pass by payload, rather than URL, if this is meant to be a list.

Reply:
I thought payloads weren't possible in get() functions?

Reply:
Ah sorry. I meant self.request.get().

Reply:
Done